presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2136,6 +2136,58 @@ public void testCommentTable()
         assertUpdate("DROP TABLE test_comment_table");
     }
 
+    @Test
+    public void testCreateTableWithHeaderAndFooter()
+    {
+        @Language("SQL") String createTableSql = format("" +
+                        "CREATE TABLE %s.%s.test_table_skip_header_footer (\n" +
+                        "   name varchar\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   format = 'TEXTFILE',\n" +
+                        "   skip_footer_count = 1,\n" +
+                        "   skip_header_count = 1\n" +
+                        ")",
+                getSession().getCatalog().get(),
+                getSession().getSchema().get());
+
+        assertUpdate(createTableSql);
+
+        MaterializedResult actual = computeActual("SHOW CREATE TABLE test_table_skip_header_footer");
+        assertEquals(actual.getOnlyValue(), createTableSql);
+
+        assertUpdate("INSERT INTO test_table_skip_header_footer VALUES ('header'), ('value'), ('footer')", 3);
+        assertQuery("SELECT * FROM test_table_skip_header_footer", "VALUES ('value')");
+
+        assertUpdate("DROP TABLE test_table_skip_header_footer");
+    }
+
+    @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = "Cannot specify skip_header_count table property for storage format: ORC")
```

Comment:
nit prefer `org.assertj.core.api.Assertions#assertThatThrownBy`

Reply:
Changed to assertThatThrownBy and added two tests.